demos/python_demos/human_pose_estimation_demo/human_pose_estimation_demo/model.py

```diff
@@ -0,0 +1,324 @@
+                [ng.result(heatmap, name='heatmaps'),
+                 ng.result(pooled_heatmap, name='pooled_heatmaps'),
+                 ng.result(paf, name='pafs')],
+                function.get_parameters(), 'hpe')
+            self.pooled_heatmaps_blob_name = 'pooled_heatmaps'
+        else:
+            self.reorder_map = np.array([0, 15, 14, 17, 16, 5, 2, 6, 3, 7, 4, 11, 8, 12, 9, 13, 10])
+            # Just rename the outputs for more convenient postprocessing.
+            f = ng.impl.Function(
+                [ng.result(heatmap, name='heatmaps'), ng.result(paf, name='pafs')],
+                function.get_parameters(), 'hpe')
+
+        self.net = IENetwork(ng.impl.Function.to_capsule(f))
+        self.exec_net = self.ie.load_network(network=self.net, device_name=self.device, num_requests=self.max_num_requests)
+        self.requests = self.exec_net.requests
+        self.empty_requests = deque(self.requests)
+
+        self.image_blob_name = self._get_inputs(self.net)
+        self.pafs_blob_name = 'pafs'
+        self.heatmaps_blob_name = 'heatmaps'
+
+        self.num_joints = self.net.outputs[self.heatmaps_blob_name].shape[1] - 1  # The last channel is for background.
+        self.size_divisor = size_divisor
+        self.target_size = self.net.input_info[self.image_blob_name].input_data.shape[-2]
+        self.output_scale = self.target_size / self.net.outputs[self.heatmaps_blob_name].shape[-2]
+        if target_size is not None:
+            self.target_size = target_size
+
+        self.decoder = OpenPoseDecoder(num_joints=self.num_joints)
+
+    def _get_inputs(self, net):
+        image_blob_name = None
+        for blob_name, blob in net.input_info.items():
+            if len(blob.input_data.shape) == 4:
+                image_blob_name = blob_name
+            else:
+                raise RuntimeError('Unsupported {}D input layer "{}". Only 2D and 4D input layers are supported'
+                                   .format(len(blob.shape), blob_name))
+        if image_blob_name is None:
+            raise RuntimeError('Failed to identify the input for the image.')
+        return image_blob_name
+
+    @staticmethod
+    def _resize_image(frame, size):
+        h, w = frame.shape[:2]
+        scale = max(size[1] / h, size[0] / w)
+        resized_frame = cv2.resize(frame, None, fx=scale, fy=scale)
+        return resized_frame
+
+    def preprocess(self, inputs):
+        img = self._resize_image(inputs[self.image_blob_name], (self.target_size, self.target_size))
+        meta = {'original_shape': inputs[self.image_blob_name].shape,
+                'resized_shape': img.shape}
+        h, w = img.shape[:2]
+        divisor = self.size_divisor
+        if w % divisor != 0 or h % divisor != 0:
+            img = np.pad(img, ((0, (h + divisor - 1) // divisor * divisor - h),
+                               (0, (w + divisor - 1) // divisor * divisor - w),
+                               (0, 0)))
```

Comment:
Older numpy (numpy==1.16.6 for me) doesn't have default value for `mode`. @eaidova, are that old versions going to be dropped?
This is also about absence of `np.core.umath.clip()`.

Reply:
2021.2 should support numpy 1.16.3 or higher if I remember that right

Reply:
Added support for older numpy versions. Tested with 1.16.3 and 1.19.4.